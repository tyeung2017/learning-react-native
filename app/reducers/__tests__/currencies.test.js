import reducer from '../currencies';
import { getInitialConversion } from '../../actions/currencies';

it('set inital state', () => {
  expect(reducer(undefined, {})).toMatchSnapshot();
});

it('set nested data on fetch data', () => {
  expect(reducer(undefined, getInitialConversion())).toMatchSnapshot();
});

