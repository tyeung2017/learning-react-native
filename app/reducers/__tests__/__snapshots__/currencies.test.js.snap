// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`set inital state 1`] = `
Object {
  "amount": 100,
  "baseCurrency": "USD",
  "conversions": Object {},
  "error": null,
  "quoteCurrency": "GBP",
}
`;

exports[`set nested data on fetch data 1`] = `
Object {
  "amount": 100,
  "baseCurrency": "USD",
  "conversions": Object {
    "USD": Object {
      "dates": "",
      "isFetching": true,
      "rates": Object {},
    },
  },
  "error": null,
  "quoteCurrency": "GBP",
}
`;
